flowzone.yml

```diff
@@ -2135,14 +2135,15 @@ jobs:
         github_publish,
         project_types,
         versioned_source,
-        website_build
+        website_build,
       ]
     # avoid being skipped when dependencies are skipped
```

Comment:
does this mean to say "Skip this step when dependencies are skipped" ?

If this step needs those steps and one of those steps are skipped than this step will also be skipped. The previous comment is confusing to me.